google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -249,6 +249,12 @@ def bucket bucket_name
         #   does not exist. For more information, see [How to Host a Static
         #   Website
         #   ](https://cloud.google.com/storage/docs/website-configuration#step4).
+        # @param [Boolean] requester_pays Indicates that a client accessing the
```

Comment:
This method already has too many optional arguments IMO.  Would it be acceptable to set this value on the yielded bucket instead of as a method argument ?